api/src/main/java/jakarta/json/bind/spi/JsonbProvider.java

```diff
@@ -125,11 +137,11 @@ public static JsonbProvider provider() {
     public static JsonbProvider provider(final String providerName) {
         if (providerName == null) {
             throw new IllegalArgumentException();
```

Comment:
Likely unrelated but... javadoc says `throws NullPointerException if providerName is {@code null}.` but the code does `if (providerName == null) { throw new IllegalArgumentException();}` (the fix belongs to different PR...)